docs/theme/mlflow/body_postscripts.html

```diff
@@ -14,12 +14,14 @@
   {%- endfor %}
 {% endif %}
 
-<script type="text/javascript" src="/_static/js/clipboard.min.js"></script>
-<script type="text/javascript" src="/_static/js/jquery.waypoints.min.js"></script>
+<script type="text/javascript" src="{{ pathto('_static/js/clipboard.min.js', 1) }}"></script>
```

Comment:
Perhaps the least obvious API of all time, passing the number `1` to `pathto` causes it to be a relative path to the web root. Note that this doesn't become relative if you start the path with `/`, though.